rep-2004.rst

```diff
@@ -778,7 +778,7 @@ If this is too onerous, then it's recommended to split lower quality packages ou
 Reference Implementation
 ========================
 
-The `ROS 2 Developer Guide <https://index.ros.org/doc/ros2/Contributing/Developer-Guide/>`_ describes the policies we implement to achieve Quality Level 1 for ROS Core packages.
+The `ROS 2 Developer Guide <https://docs.ros.org/en/kilted/The-ROS2-Project/Contributing/Developer-Guide.html>`_ describes the policies we implement to achieve Quality Level 1 for ROS Core packages.
```

Comment:
Thanks for the PR. I would link to the Rolling version so that we don't have to update it later when Kilted goes EOL.

```suggestion
The `ROS 2 Developer Guide <https://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Developer-Guide.html>`_ describes the policies we implement to achieve Quality Level 1 for ROS Core packages.
```